app/controllers/landing_controller.rb

```diff
@@ -0,0 +1,55 @@
+class LandingController < ApplicationController
+  def index
+    @categories = ['Fashion', 'Electronics', 'Home and Kitchen']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.